lib/adoptoposs_web/templates/shared/navigation_logged_in.html.eex

```diff
@@ -20,7 +20,7 @@
       <span class="ml-2">Explore</span>
     <% end %>
 
-    <%= live_redirect "Your Projects",
+    <%= live_redirect "Your Projects (#{length(@project_interests)})",
```

Comment:
It’s on purpose that the interest count appears only in the mobile navigation, to save space/not show 2 dropdowns, right? I’m not sure whether the purpose of the number might a bit misleading, I guess I’d think there is N projects, not N interests. Maybe it’s just a design issue, and putting the number in a colored background circle might make it clearer and disconnect it a bit from the "Your Projects" text.

What do you think about putting the count next to the bell icon, too?

Reply:
A colored background circle like a badge sounds like a good idea. We can put the total number of projects and the total number of project interests.

Example,
Your projects (10) (5)

> Count next to the bell icon

I try to put it up first.